src/app/views/browse/directives/browseGrid.js

```diff
@@ -609,7 +609,7 @@
         {
           name: 'civic_actionability_score',
           width: '13%',
-          displayName: 'Actionability Score',
+          displayName: 'Evidence Score',
```

Comment:
Should this be `Variant Evidence Score`?

Reply:
The column name is too narrow to fit the entire string, and since the page this is displayed on has an activated 'Variant' tab and 'Variant' in the header title I figured its relationship to variants would be clear.